### DISCUSSION TITLE
PCA9685 oscillator frequency

### DISCUSSION THREAD

**Discussion by sjgcornish** [ArminJo/ServoEasing#68] [General]
Although the nominal frequency is 25MHz, it is not very accurate. Adafruit have a setOscillatorFrequency(actual_frequency) command. In my case the frequency is 26.2Mhz. Does servoEasing have an equivalent command

**Comment by ArminJo** [comment]
This is a valid request 🥇 I implemented if by allowing to set the macro/symbol PCA9685_ACTUAL_CLOCK_FREQUENCY. See readme 😀 .

**Reply by sjgcornish** [reply]
Many thanks for the quick response that worked nicely.
